cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/AbstractQueueAdmin.java

```diff
@@ -79,43 +60,32 @@ public static String getFlowName(String queueTableName) {
     return parts[parts.length - 1];
   }
 
+  public TableId getConfigTableId(QueueName queueName) {
+    return getConfigTableId(queueName.getFirstComponent());
+  }
+
+  public TableId getConfigTableId(String namespace) {
+    return TableId.from(root, namespace, unqualifiedConfigTableNameSuffix);
```

Comment:
TableId should be aware of "root" prefix. It should be a simple POJO that has namespace and name only.